pkg/osbuild/bootc_install_to_filesystem_stage_test.go

```diff
@@ -0,0 +1,82 @@
+package osbuild_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/osbuild/images/pkg/osbuild"
+)
+
+func TestBootcInstallToFilesystemStageNewHappy(t *testing.T) {
+	devices := makeOsbuildDevices("dev-/", "dev-/boot", "dev-/boot/efi")
```

Comment:
What is this `dev-` and `mnt-` notation?

Reply:
This is indeed not very intuitive. Those setup the test device names for the mounts. I changed this now to:
```go
	devices := makeOsbuildDevices("dev-for-/", "dev-for-/boot", "dev-for-/boot/efi")
	mounts := makeOsbuildMounts("/", "/boot", "/boot/efi")
```
which hopefully makes it a bit clearer that the devices created are "for" the given mount point. Ideas how to make this nicer are very welcome, I'm still not 100% happy.